src/loaders/bitmapFontParser.js

```diff
@@ -9,11 +9,11 @@ import { BitmapText } from '../extras';
  * @function parseBitmapFontData
  * @memberof PIXI.loaders
  * @param {PIXI.loaders.Resource} resource - Loader resource.
- * @param {PIXI.Texture} texture - Reference to texture.
+ * @param {PIXI.Texture[]} textures - List of textures for each page.
  */
-export function parse(resource, texture)
+export function parse(resource, textures)
 {
-    resource.bitmapFont = BitmapText.registerFont(resource.data, texture);
+    resource.bitmapFont = BitmapText.registerFont(resource.data, textures);
 }
 
 export default function ()
```

Comment:
Do you think it is possible to add simple test for this loader?

Just passing multiple textures and check they are being loaded.

Reply:
I agree, a test is required for new functionality like this.